include/wx/valtext.h

```diff
@@ -72,6 +72,7 @@ class WXDLLIMPEXP_CORE wxTextValidator: public wxValidator
 
     // Filter keystrokes
     void OnChar(wxKeyEvent& event);
+    void OnValidation(wxValidationStatusEvent& event);
```

Comment:
Yes, and i reduced it to just

```
void wxTextValidator::OnValidationError(wxValidationStatusEvent& event)
{
    m_validatorWindow->SetFocus();
    wxMessageBox(event.GetErrorMessage(), _("Validation conflict"),
                 wxOK | wxICON_EXCLAMATION, NULL);
}

```
with the check for empty error message removed, as it might hide mistakes in user code when he/she forgot to set the error message and pass empty string instead.